Automattic/o2
# Add wp-prettier and eslint

Bring modern WP standards over.

Props @oskosk and @mirka.

### How To Test

**Note:** Make sure you don't have prettier installed globally

1. Check out this branch
2. Run `npm i`. 
3. Run `npx prettier somefile.js --write` and confirm it formats as expected and taking in account `.prettierrc` config
4. Relaunch VScode standing on this dir, save a file and confirm it saves with proper wp-prettier + .prettierrc formatting .
